### ISSUE TITLE
Wrong default color for TextInput's placeholder when multiline is true

### ISSUE THREAD

**Issue by srhb18** [facebook/react-native#14203]
ONLY IN iOS

```
<TextInput
          ref="description"
          style={styles.feedBackMessage}
          autoFocus={true}
          multiline={true}
          textAlignVertical={'top'}
          placeholder="This is a placeholder"
          underlineColorAndroid="transparent"
          onChangeText={(text) => this.setState({description: text})}
        />
```
The placeholder's color is expected grey but found black when multiline is true.

Version: 0.44
When multiline is true:
![image](https://cloud.githubusercontent.com/assets/1941010/26484434/af93d7c2-4224-11e7-9f5b-7acf3bd59044.png)

When multiline is false:
![image](https://cloud.githubusercontent.com/assets/1941010/26484448/c238395e-4224-11e7-9e6d-5bd72b04e4dc.png)

**Comment by ShylahChiera** [comment]
I'm having the same issue, found any solutions??

**Comment by hramos** [comment]
As far as I can tell from the [docs](http://facebook.github.io/react-native/docs/textinput.html), this is working as expected. Can you include the actual styles you are applying here? Ideally, can you reproduce this using [Snack](http://snack.expo.io)?

**Comment by shergin** [comment]
@srhb18 Thank you so much for bug report!
I already fixed it in my local branch a week ago and I plan to land it very soon.

**Comment by shergin** [comment]
Was fixed in 1018cc8ceb8804c1f206367877cee5267423ead2.
